cmd/controller/run.go

```diff
@@ -164,7 +164,7 @@ func Run(opts Options, runLog logr.Logger) {
 	{ // add controller for PackageInstall
 		pkgInstallCtrlOpts := controller.Options{
 			Reconciler:              NewPackageInstallReconciler(kcClient, pkgClient, coreClient, runLog.WithName("ipr")),
-			MaxConcurrentReconciles: opts.Concurrency,
+			MaxConcurrentReconciles: 1,
```

Comment:
opts.Concurrency is used for App CRs. probably was copy pasted for PackageInstalls.

> Wouldn't this be a breaking change if we remove this?

im not sure i see what it breaks.

Reply:
Are we treating this as a bug then? My thought is that this is a configurable property that is now being made unconfigurable and that is what is considered breaking.

Reply:
> i should have added to the comment: currently it seems that multiple items will get reconciled in parallel. that i think also includes same items which would in theory introduce conflicts in updates. i just set it to 1 explicitly (that's actually controller runtime default afaik) because unlike App reconciler, reconciling PackageInstall should take millis/seconds.

This is considered not breaking in follow up convo. Closing this out.